packages/types/src/config.ts

```diff
@@ -217,6 +223,21 @@ export interface DrawingsOptions {
   syncAll?: boolean
 }
 
+export interface PresenterOptions {
+  /**
+   * Restrict access to presenter mode
+   *
+   * @default true
+   */
+  isPrivate?: boolean
```

Comment:
Maybe just `private`? And I would prefer to have it default to `false` as you already need to explicitly pass `--remote` flag.

Or more over, maybe we could unify this with `token` to a single entity called `password`. And we could have the cli to be `--remote mypassowrd`.

By that, we could have a basic auth header to prompt for passing the password when visiting the route. (Or just have it in the UI)

Reply:
It is false by `default` in fact, I just forgot to update the default value in that (it has been fixed in the last commit).
I agree that this feature is only useful when having the `--remote` option, so changing to `--password` makes sense.
In that case I will remove this feature on the static build part until we have a solution to make the `--remote` option work..
I personally like the fact of having the token/password in the URL. This way you can just start the server, click on the link and avoid the additional step of typing the password.
But yes maybe we can add a prompt to enter the password in case it is not present in the URL.

Reply:
We could go both ways. Could read from the URL, and when it's not presented we show and UI for input.